Xamarin.PropertyEditing.Windows/NumericUpDownControl.cs

```diff
@@ -62,6 +62,11 @@ internal abstract class NumericUpDownControl<T>
 		: Control
 		where T : struct, IComparable<T>
 	{
+		static NumericUpDownControl ()
+		{
+			FocusableProperty.OverrideMetadata (typeof (NumericUpDownControl<T>), new FrameworkPropertyMetadata (false));
```

Comment:
I could have forgotten my focusable rules so forgive me if i get it wrong ;) The numeric control itself should not be focusable or a tab stop because it does nothing in and of itself. The `up` button, `down` button and the entry textbox contained inside it should be focusable/tabstops though